bindings/2.204/GeometryDash.bro

```diff
@@ -11746,7 +11746,7 @@ class RewardsPage : FLAlertLayer, FLAlertLayerProtocol, GJRewardDelegate {
 	virtual bool init() = win 0x2f67f0;
 	virtual void registerWithTouchDispatcher();
 	virtual void keyBackClicked();
-	virtual void show();
+	virtual void show() = win 0xf17f0;
```

Comment:
this is most likely wrong, `0xf17f0` is somewhere in the middle of another function.

the real `RewardsPage::show` is merged with `ProfilePage::show` i believe, so it would have the same address as that one

Reply:
Oh yeah you're right. 

In that case, the merged address is `0x2f17f0`, but I tested it in a mod that opens this popup and both works (`0xf17f0` and `0x2f17f0`)